static/js/popovers.js

```diff
@@ -853,25 +865,23 @@ exports.register_click_handlers = function () {
         e.preventDefault();
     });
 
-    $("body").on("click", ".set_away_status", (e) => {
-        exports.hide_all();
-        user_status.server_set_away();
-        e.stopPropagation();
-        e.preventDefault();
-    });
-
-    $("body").on("click", ".revoke_away_status", (e) => {
-        exports.hide_all();
-        user_status.server_revoke_away();
-        e.stopPropagation();
-        e.preventDefault();
+    $("body").on("click", ".status_info", (e) => {
+        $(".status_info").on("change", () => {
+            const option = $(".status_info option:selected").val();
```

Comment:
We don't need to declare a variable if it's only used once in an if condition. Also instead of `$(".status_info option:selected").val()` you should be able to do `$(this).val()`, so the if condition could probably just be `$(this).val() === "inactive"`.

Reply:
This is not resolved.